group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroup.java

```diff
@@ -998,4 +1055,93 @@ public ConsumerGroupDescribeResponseData.DescribedGroup asDescribedGroup(
         );
         return describedGroup;
     }
+
+
+    /**
+     * Set the attributes of the consumer group according to a classic group.
+     * Add the records for creating and updating the consumer group.
+     *
+     * @param classicGroup      The converted classic group.
+     * @param records           The list to which the new records are added.
+     */
+    public void fromClassicGroup(
```

Comment:
The order of the records are still incorrect. I let you go back to my previous comment about it. We can also discuss it offline if needed.